### DISCUSSION TITLE
Form Error Exist, new UI

### DISCUSSION THREAD

**Discussion by Century1026** [inventree/InvenTree#7240] [Q&A]
When counting stocks in new UI, an error occurred. But this never happen in old UI.
![image](https://github.com/inventree/InvenTree/assets/106154819/101c6ca1-7968-47c2-9a1d-1398bea95c4f)

**Comment by SchrodingersGat** [comment]
Thanks for reporting! Please note that the new interface is under heavy development, so feedback on these errors is critical :)

Can you please provide information on the error message (it should be present in the "network" tab of your browser console).

**Comment by Century1026** [comment]
this one?
![image](https://github.com/inventree/InvenTree/assets/106154819/662ce956-3058-4144-b8c8-59409d5c4867)

**Reply by SchrodingersGat** [reply]
@Century1026 yep, thanks. Will have to look into this!
